sdk/communication/azure-communication-phonenumbers/test/_shared/testcase.py

```diff
@@ -25,14 +25,14 @@ def process_response(self, response):
                     if isinstance(item, str):
                         body["phoneNumbers"] = [self._replacement]
                         break
-                    elif "phoneNumber" in item:
```

Comment:
why we change from elif to if here?

Reply:
I wanted to sanitize both values. Having an elif only sanitized the phone number or the id, but not both at the same time.